.github/workflows/check_lint_build_release.yaml

```diff
@@ -207,15 +210,18 @@ jobs:
       - name: Create zip files for releases.drivechain.info
         run: |
           shopt -s extglob
-          mv bip300301_enforcer-+([0-9]).+([0-9]).+([0-9])-x86_64-apple-darwin bip300301-enforcer-latest-x86_64-apple-darwin
-          zip -r bip300301-enforcer-latest-x86_64-apple-darwin.zip \
-            bip300301-enforcer-latest-x86_64-apple-darwin
-          mv bip300301_enforcer-+([0-9]).+([0-9]).+([0-9])-x86_64-pc-windows-gnu bip300301-enforcer-latest-x86_64-pc-windows-gnu
-          zip -r bip300301-enforcer-latest-x86_64-pc-windows-gnu.zip \
-            bip300301-enforcer-latest-x86_64-pc-windows-gnu
-          mv bip300301_enforcer-+([0-9]).+([0-9]).+([0-9])-x86_64-unknown-linux-gnu bip300301-enforcer-latest-x86_64-unknown-linux-gnu
-          zip -r bip300301-enforcer-latest-x86_64-unknown-linux-gnu.zip \
-            bip300301-enforcer-latest-x86_64-unknown-linux-gnu \
+          # Darwin
+          if [ -f bip300301-enforcer-latest-x86_64-apple-darwin ]; then
+            zip -r bip300301-enforcer-latest-x86_64-apple-darwin.zip bip300301-enforcer-latest-x86_64-apple-darwin
+          fi
+          # Windows
+          if [ -f bip300301-enforcer-latest-x86_64-pc-windows-gnu.exe ]; then
+            zip -r bip300301-enforcer-latest-x86_64-pc-windows-gnu.zip bip300301-enforcer-latest-x86_64-pc-windows-gnu.exe
+          fi
+          # Linux
+          if [ -f bip300301-enforcer-latest-x86_64-unknown-linux-gnu ]; then
+            zip -r bip300301-enforcer-latest-x86_64-unknown-linux-gnu.zip bip300301-enforcer-latest-x86_64-unknown-linux-gnu
+          fi
```

Comment:
We should expect all files here. Any reason in particular for checking for the existence first? Could lead to silently swallowing issues where we're not uploading all three as expected